web-marketplace/src/lib/ecocredit/api.ts

```diff
@@ -199,31 +198,6 @@ export const getEcocreditsForAccount = async ({
   }
 };
 
-export const getEcocreditTxs = async (): Promise<TxResponse[]> => {
```

Comment:
Both `getEcocreditTxs` and  `queryEcoClassInfo` were no longer used so I removed them as well.